frameaccurate/index.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<!--
+*
+* Copyright (C) 2018, Bitmovin Inc, All Rights Reserved
+*
+* This source code and its use and distribution, is subject to the terms
+* and conditions of the applicable license agreement.
+*
+-->
+<html lang="en">
+<head>
+    <title>Bitmovin Demo</title>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
+    <link rel="icon" type="image/png" href="bit-fav.png">
+    <!-- Bitmovin player -->
+    <script type="text/javascript" src="https://bitmovin-a.akamaihd.net/bitmovin-player/stable/7/bitmovinplayer.js"></script>
+    <!-- Bootstrap 3.3.7 -->
+    <link rel="stylesheet" href="css/bootstrap.min.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/bitmovin-demo.css">
+    <link rel="stylesheet" href="css/frameaccurate-demo.css">
+</head>
+<body>
+<div id="wrapper">
+
+    <div id="banner">
+        <div class="logo"><a href="https://bitmovin.com"><img src="bitmovin-logo.png"></a></div>
+        <div class="title"><h1 class="bitmovin-headline">Frame Accurate Seeking</h1></div>
+        <div class="clearfix"></div>
+    </div>
+
+    <div class="container-fluid">
+        <div class="content">
+            <div class="row">
+                <div id="test-asset-wrapper" class="col-lg-12 text-center">
+                    <div>
+                        <h3>Test assets:</h3>
+                        <select id="test-assets"></select>
```

Comment:
Please select one asset always and remove the selection for the sake of simplicity.